edge-agent/docker/linux/amd64/Dockerfile

```diff
@@ -9,7 +9,7 @@ ARG EXE_DIR=.
 RUN apk update && \
     apk add --no-cache snappy
 
-# create a user to allow agent to optionally run as non-root
+# Create a user to allow installing the trustbundle under a user context
```

Comment:
What do you mean by installing trust bundle under a user context? Where does the trustbundle get installed?

Reply:
We set the X509store location to `CurrentUser` - which creates the store in the current user's directory. If agent is running as root, it uses `/root` as the directory, this creates a user for installing. For Edge on K8s, we can run Agent as non-root, so we create this user for a place to put the X509 store.